### DISCUSSION TITLE
Node.js support

### DISCUSSION THREAD

**Discussion by liatbor** [eclipse-theia/theia#10735] [Q&A]
Hi,
Which Node.js versions are supported in Theia 1.19?
Thanks,
Liat

**Comment by vince-fugnitto** [comment]
@liatbor thank you for the discussion, looking at the `v1.19.0` tag we can see the [prerequisites](https://github.com/eclipse-theia/theia/blob/v1.19.0/doc/Developing.md#prerequisites):

https://github.com/eclipse-theia/theia/blob/e1da7a1b035e84966f1692783c61e768ffe013f1/package.json#L7

I believe `v12` would give you the best compatibility, anything newer may require you to pass `--ignore-engines` when building, or may have incompatibilities with electron.
